segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/AppendProcessorTest.java

```diff
@@ -1014,6 +1048,23 @@ public void testAppendAfterSealThrows() throws Exception {
         verifyNoMoreInteractions(store);
     }
 
+
+    private static ServiceBuilderConfig getBuilderConfig() {
+        return ServiceBuilderConfig
+                .builder()
+                .include(ServiceConfig.builder()
+                        .with(ServiceConfig.CONTAINER_COUNT, 1)
+                        .with(ServiceConfig.THREAD_POOL_SIZE, 3)
+                        .with(ServiceConfig.LISTENING_PORT, TestUtils.getAvailableListenPort()))
+                .build();
+    }
+
+    private static ServiceBuilder newInlineExecutionInMemoryBuilder(ServiceBuilderConfig config) {
```

Comment:
Every other test in this class mocks the segment store. I strongly recommend we continue following that pattern. Here, you're just testing the `AppendProcessor` and need not start the whole segment store for that.

Reply:
I was trying to mock it out first but the connection would always fail to verify the `TransientSegmentCreated` was sent.